v1.3.0

AlbertoElias opened the pull request:
Updates Wallet version

### How to test?

<!-- What steps in order should someone run to test -->

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Respect code style and lint
- [ ] Update documentation (if needed)

elias-garcia commented:
Why 1.3.0? I think that it doesn't introduce any new feature right? I've already update it to 1.2.1 yesterday

AlbertoElias commented:
It's a pretty big underlying change that can come with many bugs. We've also fixed several bugs and added multiple features like transaction flow improvements.
